Dockerfile

```diff
@@ -9,10 +9,11 @@ COPY ./ .
 RUN CGO_ENABLED=0 go build
 
 FROM scratch
+USER 1000
```

Comment:
I like the general idea to move away from root user, I can understand that.
what I do not like:
* there are no guarantees that user `1000` exists on the host, so the mapping from the container is suspicious, it works, but I do not like it
* the pwent is not present for the uid `1000`, which I do not like, we should run containers that are fully configured from the perspecitve of the process running in them

Question:
* could we run the container with `--user` (or `user:`) and leave the Dockerfile alone?